parity/run.rs

```diff
@@ -721,11 +720,7 @@ fn execute_impl<Cr, Rr>(cmd: RunCmd, logger: Arc<RotatingLogger>, on_client_rq:
 
 	// set up dependencies for rpc servers
 	let rpc_stats = Arc::new(informant::RpcStats::default());
-	let secret_store = match cmd.public_node {
-		true => None,
-		false => Some(account_provider.clone())
-	};
-
+	let secret_store = Some(account_provider.clone());
```

Comment:
Is it possible now to remove the `Option` from `secret_store` in the rpc dependencies? Then could remove these unwraps e.g. https://github.com/paritytech/parity/blob/master/rpc/src/v1/impls/signing_unsafe.rs#L111.